build.rs

```diff
@@ -13,4 +13,37 @@ fn main() {
     {
         println!("cargo::rustc-cfg=feature=\"include-ui\"");
     }
+    if env::var_os("CARGO_CFG_WINDOWS").is_some() {
+        match std::fs::File::open("assets/icon.png") {
+            Ok(file) => {
+                println!("This is windows, creating icon.");
+                // Create a new, empty icon collection
+                let mut icon_dir = ico::IconDir::new(ico::ResourceType::Icon);
+                // Read a PNG file from disk and add it to the collection
+                let file =
+                    std::fs::File::open("assets/icon.png").expect("Could not find assets/icon.png");
```

Comment:
You can remove these lines, the file is already opened in the `match` above. This will also prevent the Clippy warning.